templates/address-space-controller/020-ConfigMap-address-space-definitions.yaml

```diff
@@ -1415,3 +1454,11 @@ data:
     - description: Broker address full policy
       name: BROKER_ADDRESS_FULL_POLICY
       value: FAIL
+    - description: The name of the broker
+      displayName: AMQ Name
+      name: AMQ_NAME
+      value: broker
+    - description: Home directory to for the broker instance
+      displayName: Home
+      name: HOME
+      value: /home/artemis
```

Comment:
I'm not sure it makes sense to templatize these values. I think instead we should just set them in the pod container env.

Reply:
now removed